.github/ISSUE_TEMPLATE/feature_proposal.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Feature proposal
+about: Propose a great new feature for Mitiq
+title: ''
+labels: 'enhancement'
+assignees: ''
+
+---
+## Pre-Request Checklist
+
+- [ ] I checked to make sure that this feature has not already been requested
+- [ ] I or my organization would be willing to help implement and/or test this.
+- [ ] This is **not** a breaking change to Mitiq.
```

Comment:
I might remove this - some features could be breaking changes, no? and some quantum-but-not-software people may not be familiar with this terminology.

Reply:
I guess my thought here is not all need to be checked to be valid, its just more so we can gauge quickly how much work is involved. I take the point that breaking change might be not familiar outside of SW dev.